protos/in_toto_attestation/predicates/vuln/v0.1/vuln.proto

```diff
@@ -0,0 +1,46 @@
+syntax = "proto3";
+
+package in_toto_attestation.predicates.vuln.v01;
```

Comment:
Should this be `vulns` instead? 

[Spec](https://github.com/in-toto/attestation/blob/main/spec/predicates/vuln.md) seems to be `vulns`, i do know at least 1 implementation already doing that, so my plead would be to go with vulns for `v0.1` :)

Reply:
@lumjjb I've made some changes about it.